platforms/android/src/org/crosswalk/engine/XWalkBridgeEngine.java

```diff
@@ -128,6 +128,12 @@ static public boolean shouldMakeXwalkWebView(Context context) {
         }
         
         PackageManager packageManager = context.getPackageManager();
+        try {
+            packageManager.getPackageInfo("\"com.google.android.webview\"", 0);
+        } catch (PackageManager.NameNotFoundException e) {
+            XWalkBridgeEngine.cachedShouldMakeXwalkWebView = true;
+            return true;
+        };
         int enabledSetting = packageManager.getApplicationEnabledSetting("com.google.android.webview");
```

Comment:
there is an uncaught exception here which crashes the app if the device does not have com.google.android.webview installed.